tests/api/data/e2e/alerts/active__status=actual&area=AL.json

```diff
@@ -47,7 +47,7 @@
         "effective": "2024-01-08T13:48:00-06:00",
         "onset": "date:now -1 hour",
         "expires": "date:now +1 hour",
-        "ends": "date:now +0 hour",
+        "ends": "date:now -1 minute",
```

Comment:
This was working entirely by accident. In our tests, we compared it to "now," and the only reason the alert ending time was after "now" is because there were a few milliseconds between when we initiated the API request and when our proxy parsed these tokens into a date. This change makes it explicit and removes the narrowly-timed, almost race condition.